vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc

```diff
@@ -13,9 +13,20 @@
 
 namespace confirmations {
 
-namespace {
+bool ShouldLogHeader(const std::string& header) {
```

Comment:
for consistency with other ads naming, can we please use `ShouldExcludeHeader`

Reply:
fixed